ffs/rpc/rpc.go

```diff
@@ -507,8 +480,8 @@ func (s *RPC) Close(ctx context.Context, req *CloseRequest) (*CloseResponse, err
 	return &CloseResponse{}, nil
 }
 
-// AddToHot stores data in the Hot Storage so the resulting cid can be used in PushConfig.
-func (s *RPC) AddToHot(srv RPCService_AddToHotServer) error {
+// Stage stores data in the Hot Storage so the resulting cid can be used in PushStorageConfig.
```

Comment:
Maybe `s/stores/caches` as you described in the pow cli?

Reply:
👍